stable/apm-server/templates/deployment.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if eq .Values.kind "Deployment" }}
+apiVersion: extensions/v1beta1
```

Comment:
Can you please use the `apps/v1` version for Deployment. The beta versions will soon not be accepted by the Kubernetes API.